src/typescript/frontend/src/components/pages/emoji-picker/EmojiPicker.tsx

```diff
@@ -198,7 +198,7 @@ export default function EmojiPicker(props: HTMLAttributes<HTMLDivElement>) {
           onClickOutside={onClickOutside}
           perLine={8}
           // TODO: Use this instead of the current "not allowed" emoji we're using..?
-          // exceptEmojis={["🔥", "🚀", "🌙", "🌟", "🎉", "🎊", "🎈", "🎁", "🎆", "🎇"]}
+          exceptEmojis={["🔥", "🚀", "🌙", "🌟", "🎉", "🎊", "🎈", "🎁", "🎆", "🎇"]}
```

Comment:
Didn't mean to add. It w ok nt do anything anyway because it used text IDS, not emojis. Fixed in next PR if not this one